pkg/config/compatibility.go

```diff
@@ -85,7 +85,7 @@ func (i *CUDABaseImage) ImageTag() string {
 var cudaBaseImagesData []byte
 var CUDABaseImages []CUDABaseImage
 
-//go:generate go run ../../tools/compatgen/main.go tensorflow -o tf_compatibility_matrix.json
+//go:generate go run ../../tools/compatgen/main.go tensorflow --min-cuda-version 11 -o tf_compatibility_matrix.json
```

Comment:
This is a bit magical but AFAIK there's no good way to pass a variable from the package to `go generate`

Reply:
Setting a default value for that flag should do the trick. See https://github.com/replicate/cog/pull/1715/files#r1628418365.